ark/phenotyping/som_utils.py

```diff
@@ -562,15 +560,16 @@ def create_fov_pixel_data(fov, channels, img_data, seg_labels,
     pixel_mat = normalize_rows(pixel_mat, channels, seg_labels is not None)
 
     # subset the pixel matrix for training
-    pixel_mat_subset = pixel_mat.sample(frac=subset_proportion, random_state=seed)
+    pixel_mat_subset = pixel_mat.sample(frac=subset_proportion)
```

Comment:
Is there a reason we got rid of the random seed?

Reply:
We got rid of it because `random_state` in `pandas` selects the same rows indices for each `fov` when passed directly as an argument to `sample`. So we have to preset the seed beforehand so there's stochasticity in the row selection but the results are still replicable.